webapp/graphite/tags/utils.py

```diff
@@ -0,0 +1,94 @@
+"""Utility functions for tag databases."""
+import abc
+
+try:
+    from importlib import import_module
+except ImportError:  # python < 2.7 compatibility
+    from django.utils.importlib import import_module
+
+
+class TaggedSeries(object):
+  @classmethod
+  def parse(cls, path):
```

Comment:
I went this way to keep the API nice and simple, I doubt the overhead of creating the object will amount to much.